components/button.css

```diff
@@ -0,0 +1,128 @@
+@import url(/stylesheets/style.css);
+
+
+.page-container{
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+
+.container{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.5rem;
+}
+
+.btn{
+    padding: 0.8rem 2rem;
+    border-radius: 6px;
+    border: none;
+    outline: none;
+    cursor: pointer;
+
+}
+
+.btn-primary{
+    color: white;
+    background-color: var(--primary-100);
+}
+
+.btn-secondary{
+    color: white;
+    background-color: var(--secondary-100);
+}
+
+.btn-danger{
+    color: white;
+    background-color: var(--danger);
+}
+
+.btn-success{
+    color: white;
+    background-color: var(--success);
+}
+
+.btn-warning{
+    color: white;
+    background-color: var(--warning);
+}
+
+.btn-link{
+    text-decoration: none;
+    color: white;
+    background-color: var(--link-color);
+    padding: 0.6rem 1rem;
+}
+
+.outlined-primary{
+    color: var(--primary-100);
+    background-color: white;
+    border: 1px solid ;
+}
+
+.outlined-secondary{
+    color: var(--secondary-100);
+    background-color: white;
+    border: 1px solid ;
+}
+
+.outlined-danger{
+    color: var(--danger);
+    background-color: white;
+    border: 1px solid ;
+}
+
+.outlined-success{
+    color: var(--success);
+    background-color: white;
+    border: 1px solid ;
+}
+
+.outlined-warning{
+    color: var(--warning);
+    background-color: white;
+    border: 1px solid ;
+}
+
+.outlined-link{
+    text-decoration: none;
+    color: var(--link-color);
+    background-color: white;
+    padding: 0.6rem 1rem;
+    border: 1px solid ;
+}
+
+.icon{
+    color: var(--primary-100);
+    outline: none;
+    border: none;
+    background-color: white;
+    font-size: large;
+    
+}
+
+.fas{
+    font-size: 2rem;
+    color: white;
+}
+
+.btn-rounded{
+    height: 3rem;
+    width: 3rem;
+    border-radius: 50%;
+    border: none;
+    background-color: var(--primary-100);
+}
+
+
+
+:root{
```

Comment:
variable declarations in root should be on the bottom. Its good practice to declare these on top